### DISCUSSION TITLE
Can't verify my academic affiliation due to the earthquakes in the past months

### DISCUSSION THREAD

**Discussion by sumc-hub** [community/community#51328] [GitHub Education]
Hi,
Due to frequenting earthquakes in Turkey, the Council of Higher Education has decided to continue the spring semester's education online. Because of this I continue my education at a distance from my university but this is causing a verify issue. There is an official press release for this but I don't know if this press release enough to confirm my student status.

**Comment by samus-aran** [comment]
Hi @sumc-hub 👋 

Sorry to hear of the earthquakes, hope you and your family are safe and well!

Here are a couple of tips to help with resubmitting your application:

1. For remote student applicants: please ensure that your evidence indicates that you are a remote student. We would suggest that due to the circumstances that you request from your institution a document indicating that for the next semester that it will be remote or distant learning. Include this document alongside your current evidence.
2. If you have recently updated your profile name but are still receiving a message asking to update it please try logging out of GitHub and log back in to re-apply. 

Let us know how you get on 👍
